ligo/src/proposal.mligo

```diff
@@ -158,9 +155,13 @@ let check_vote_limit_reached
           ("MAX_VOTES_REACHED", ()) : vote_param)
     else vote_param
 
-let vote(votes, config, store : vote_param_permited list * config * storage): return =
+// Note: we require this because we want to put the entrypoint in a lamba and so
+// we need the contract address to be provided (can't use 'SELF_ADDRESS').
```

Comment:
> we need the contract address to be provided (can't use 'SELF_ADDRESS').

But why does the Edo document say it is allowed in Lambda?

>Instruction SELF_ADDRESS equivalent to SELF; ADDRESS but allowed in lambdas

https://tezos.gitlab.io/protocols/008_edo.html#michelson

Reply:
That's correct, but currently LIGO converts it to `SELF; ADDRESS` and so that won't work.

I think we should update this when LIGO is updated after `edo` activation, in the meanwhile I will update this comment.